app/steps/ui/declaration/index.js

```diff
@@ -190,6 +192,13 @@ class Declaration extends ValidationStep {
         delete ctx.invitesSent;
         return [ctx, formdata];
     }
+
+    renderPage(res, html) {
+        const formdata = res.req.session.form;
+        formdata.legalDeclaration = legalDocumentJSONObjBuilder.build(formdata, html);
```

Comment:
Do we need to know what value `legalDeclaration` has after the user has finished the application? For example if they log back in again or for a backend service?

If not and this value is just used for the current session then it would be better to store in the session rather than in the form data and persisting it to the db.